API Generator: Add missing `scope` argument and filter to `<entity>BySlug` query (#1643)
vivid-planet/comet: .changeset/three-eagles-cheat.md

---

> "@comet/cms-api": patch

Reviewer: ```suggestion
"@comet/cms-api": minor
```

IMO this is a feature, what do you think?

Author: as I talked with @nsams he thougt this is a bug, minor is ok for me?

Reviewer 2: Whatever

Author: @johnnyomair what do you want? minor or patch?

Reviewer: Patch is fine for me. Will change the changeset accordingly.

---

> api-generator: add scope argument and filter for slug query

Reviewer: ```suggestion
API Generator: Support `scope` argument and filter in `<entity>BySlug` query
```
